src/runtime/graph/graph_runtime.cc

```diff
@@ -123,9 +123,20 @@ int GraphRuntime::NumInputs() const {
  */
 std::string GraphRuntime::GetInputName(int index) const {
   CHECK_LT(static_cast<size_t>(index), input_nodes_.size())
-      << "The index of ouf of range.";
+      << "The index is out of range.";
   return nodes_[input_nodes_[index]].name;
 }
+/*!
+ * \brief Get the number of the index-th input.
```

Comment:
I didn't notice this last time, but the description is not updated. Same to the one in the .h file.

Reply:
fixed